src/universal/redux/storageMerger.js

```diff
@@ -0,0 +1,21 @@
+import jwtDecode from 'jwt-decode';
+
+export default function merger(initialState, persistedState) {
+  const whitelist = new Set(['authToken', 'cashay', 'notifications']);
```

Comment:
we'll have to update this for every new duck that we create. in the future, we could turn this into a `blacklist` depending on the result of https://github.com/reactjs/redux/issues/1792